ocaml-lsp-server/src/dune

```diff
@@ -5,7 +5,6 @@
  (libraries cmdliner dune-build-info fiber fiber_unix jsonrpc lsp lsp_fiber
    merlin.analysis merlin.kernel merlin.merlin_utils merlin.parsing
    merlin.query_commands merlin.query_protocol merlin.specific merlin.typing
-   merlin.utils octavius omd ppx_yojson_conv_lib result stdune yojson
-   dune_rpc))
+   merlin.utils octavius omd ppx_yojson_conv_lib result stdune yojson))
```

Comment:
Was this change accidental? We do in fact need to depend on dune_rpc

Reply:
Yes, I dropped that commit.